misc/object-conditions/basics.rst

```diff
@@ -468,6 +487,12 @@ Date & Time field
           Year(s).*
       * | *has changed*
         | *The field has been changed during a ticket update.*
+      * | *has reached*
+        | **This option is only available for Ticket's Pending time and Escalation time in Trigger conditions!**
+        | *The time set in this field was reached*
+      * | *has reached warning*
+        | **This option is only available for Ticket's Escalation time in Trigger conditions!**
+        | *The time set in this field will be reached in less than 15 minutes*
```

Comment:
Maybe instead of bolding the hints about value availability, it would be better to render them in a separate hint block. Something similar was done above in another context:

```rst
.. hint:: This field is only available for Trigger and Scheduler conditions!
```

Let's at least try it, if it fits nicely with the rest of values.

Reply:
I did try it, but it looked very strange with a big blob in a split column.

![Screenshot 2023-04-19 at 23 57 12](https://user-images.githubusercontent.com/36944/233198412-1e644d80-1457-468f-8182-bb9079098368.png)

Reply:
Want me to have a look? It's my fault so... :)